src/pages/profile.tsx

```diff
@@ -78,7 +94,84 @@ export default function Page() {
       <Input background="white" 
         value={newCity} onChange={e => setNewCity(e.target.value)} />
     </FormControl>           
+
+    <Divider margin={componentSpacing} />
+
+    <InterviewPreference />
+
     <Button onClick={handleSubmit} variant="brand">保存</Button>
     {isLoading && <Loader loadingText='保存中...'/>}
   </VStack>;
 }
+
+function InterviewPreference({
+
+} : {
+  
+}) {
+  const [date, setDate] = useState(oneMonthDate);
+  const [limit, setLimit] = useState<Limit | undefined>(undefined);
+
+  return <>
+    <FormControl>
+      <FormLabel>面试官偏好</FormLabel>
+      <Checkbox sx={{ '.chakra-checkbox__control': { bg: 'white' } }}>
```

Comment:
actually, don't alter styles here. do it in src/theme. Create checkboxes.ts under this folder

Reply:
do the same for radio background colors, add `radio.ts` under src/theme